todo-app/src/Todo.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Description from './Description';
+import Deadline from './Deadline';
+
+
+export default class Todo extends React.Component{
+   
+
+    render(){
+
+        const { todo, handleDoneToggle } = this.props;
+
+        return(
+            <li className="Todo">
+
+                <div className="Todo-Description">
+                   <Description task={todo.task}/>
+                </div> 
+                    
+                <div className="Todo-Deadline">
+                   <Deadline time={todo.time}/>
+                </div> 
+
+                <div>
+                   is done : <input onClick={() => handleDoneToggle(todo.id)} 
```

Comment:
Your checkbox is "uncontrolled" (do you know why?) and as such will not display a check if the item was already done in the JSON file (see https://reactjs.org/docs/uncontrolled-components.html). What would you need to change to make this checkbox controlled?